src/otelcollector/otelcol-config-extras.yml

```diff
@@ -1,2 +1,26 @@
 # extra settings to be merged into OpenTelemetry Collector configuration
 # do not delete this file
+
+exporters:
+  datadog:
+    api:
+      key: <DATADOG_API_KEY> # Replace with your Datadog API key
```

Comment:
Would prefer `DD_API_KEY`. We use that in most examples

Reply:
Thanks! Fixed.